### PR TITLE
Use highlight js

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
FYI: just pushed an `oops' commit that adds swift, obj-c, and java (for Flutter interop packages)

Will cleanup/rebase once approved

**Comment by kevmoo** [comment]
Thoughts @jcollins-g and @devoncarew ?

File: lib/resources/github.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/*
+
+github.com style (c) Vasily Polovnyov <vast@whiteants.net>
```
Comment (jcollins-g) [inline lib/resources/github.css]: Please indicate somewhere in LICENSE/README where this and other new third-party libraries included come from (including URLs) and their licensing.

File: lib/src/markdown_processor.dart [hunk 2]
```diff
@@ -797,7 +797,6 @@ class MarkdownDocument extends md.Document {
       }
 
       bool specifiesLanguage = pre.classes.isNotEmpty;
-      pre.classes.add('prettyprint');
```
Comment (jcollins-g) [inline lib/src/markdown_processor.dart]: Could you mention a bit more about how this is replaced?

File: tool/travis.sh [hunk 3]
```diff
@@ -57,5 +57,5 @@ else
   (cd testing/test_package_small; dart -c ../../bin/dartdoc.dart)
 
   # Run the tests.
-  pub run grinder test
+  pub run test
```
Comment (jcollins-g) [inline tool/travis.sh]: Why the change?

File: tool/grind.dart [hunk 4]
```diff
@@ -181,9 +180,9 @@ publish() async {
 
 @Task('Run all the tests.')
 test() {
-  // this is 5 seconds faster than `pub run test`, so
-  // using straight-up VM here
-  return Dart.runAsync('test/all.dart', vmArgs: ['--checked']);
+  // `pub run test` is a bit slower than running an `test_all.dart` script
+  // But it provides more useful output in the case of failures.
```
Comment (jcollins-g) [inline tool/grind.dart]: Could you be more specific?

File: testing/test_package/lib/fake.dart [hunk 5]
```diff
@@ -496,7 +496,7 @@ class BaseForDocComments {
   ///
   /// Reference to a top-level const in another library [incorrectDocReferenceFromEx]
   ///
-  /// Reference to prefixed-name from another lib [css.theOnlyThingInTheLibrary] xx
+  /// Reference to prefixed-name from another lib [example_prefixed.incorrectDocReferenceFromEx]
```
Comment (jcollins-g) [inline testing/test_package/lib/fake.dart]: Does css.theOnlyThingInTheLibrary still fail?

File: lib/src/markdown_processor.dart (hunk 2 above)
Comment (kevmoo) [reply lib/src/markdown_processor.dart]: `prettyprint` was used by the pretty print library to find elements to style at runtime.

`highlight.js` looks at all code blocks with `language-XYZ` – no prettyprint needed

File: tool/travis.sh (hunk 3 above)
Comment (kevmoo) [reply tool/travis.sh]: See comments on this commit: https://github.com/dart-lang/dartdoc/pull/1489/commits/90a379d9d5845616cf8b8a050c15bc4fda5e4c89

File: testing/test_package/lib/fake.dart (hunk 5 above)
Comment (kevmoo) [reply testing/test_package/lib/fake.dart]: That member was removed – but since this test was skipped, it didn't change output.

Just picked something that already existed.

File: lib/resources/github.css (hunk 1 above)
Comment (kevmoo) [reply lib/resources/github.css]: Done

File: tool/grind.dart (hunk 4 above)
Comment (kevmoo) [reply tool/grind.dart]: See comments on this commit: 90a379d

**Comment by kevmoo** [comment]
@jcollins-g here's the before/after

For the original bug, the new style is much more clean about comments vs ID selectors.

Before

screenshot_before

After

screenshot

File: testing/test_package/lib/fake.dart (hunk 5 above)
Comment (jcollins-g) [reply testing/test_package/lib/fake.dart]: The member isn't actually gone, so this test should pass (assuming we didn't have the bug it is testing for).

**Comment by kevmoo** [comment]
Sorry. Had to do a rebase fix – but went back to the original

File: test/model_test.dart [hunk 6]
```diff
@@ -461,17 +461,10 @@ void main() {
...
-              'Wait for https://github.com/dart-lang/dartdoc/issues/767 to be fixed');
-
-      // remove this test when the above test is fixed. just here to
-      // track when the behavior changes
-      test('codeifies a prefixed top-level variable an imported lib', () {
         expect(
-            docsAsHtml, contains('<code>css.theOnlyThingInTheLibrary</code>'));
+            docsAsHtml,
+            contains(
+                '<p>Reference to prefixed-name from another lib <code>css.theOnlyThingInTheLibrary</code> xx</p>'));
```
Comment (jcollins-g) [inline test/model_test.dart]: Now this has " xx" on the end?

**Comment by kevmoo** [comment]
@jcollins-g – FYI. I removed the fixed test.

In the test case, the link is still broken....but in the full output, the link is there.

Zero clue why.
